api/rosetta/balance.go

```diff
@@ -48,36 +48,42 @@ func (d *Data) Balance(ctx echo.Context) error {
 	var req BalanceRequest
 	err := ctx.Bind(&req)
 	if err != nil {
-		return echo.NewHTTPError(http.StatusBadRequest, InvalidFormat(err.Error()))
+		return echo.NewHTTPError(http.StatusBadRequest, InvalidEncoding("request does not contain valid JSON-encoded body", err))
 	}
 
 	if req.NetworkID.Blockchain == "" {
-		return echo.NewHTTPError(http.StatusBadRequest, InvalidFormat("blockchain identifier: blockchain field is empty"))
+		return echo.NewHTTPError(http.StatusBadRequest, InvalidFormat("blockchain identifier has empty blockchain field"))
```

Comment:
After reading it differently in the other files, I think the wording `network identifier` is actually better. Since the JSON field is `network_identifier`, not `blockchain_identifier`.

Reply:
I thought I fixed that everywhere, but apparently not.